HW03g.sql

```diff
@@ -0,0 +1,176 @@
+﻿--1 Выберите сотрудников, которые являются продажниками, и еще не сделали ни одной продажи.
+SELECT
+*
+FROM Application.People p
+WHERE p.IsSalesperson = 1
+AND NOT EXISTS (SELECT * FROM Sales.Orders s WHERE s.SalespersonPersonID = p.PersonID )
+
+--2. Выберите товары с минимальной ценой (подзапросом), 2 варианта подзапроса. 
+SELECT
+s.StockItemName
+FROM Warehouse.StockItems s
+WHERE s.UnitPrice <= (SELECT MIN(UnitPrice) FROM Warehouse.StockItems )
```

Comment:
Можно просто `=`
```suggestion
WHERE s.UnitPrice = (SELECT MIN(UnitPrice) FROM Warehouse.StockItems )
```

А с `ALL`, да, правильно `<=`